server/interpreter.js

```diff
@@ -2346,10 +2287,8 @@ Interpreter.prototype.unwind_ = function(type, value, label) {
   if (type === Interpreter.Completion.THROW) {
     // Log exception and stack trace.
     if (value instanceof this.Error) {
-      var name = this.getProperty(value, 'name');
-      var message = this.getProperty(value, 'message');
-      console.log('Unhandled %s: %s', name, message);
-      var stackTrace = this.getProperty(value, 'stack');
+      console.log('Unhandled %s', value.toString());
```

Comment:
I think the .toString() is redundant.

Reply:
Yes, but not in the way you think:

By default, Node's `console.log` prints a nice dump of the structure of the logged object.  The call to `toString()` was intended to avoid dumping the internal structure, so that objects would be shown as `[object Object]`.  But for some reason it doesn't work, and the output actually shows the internal anyway!

Reply:
No, never mind; I was being an idiot and looking at this code but actually testing the bit a few lines down where non-Error exceptions are dealt with.

Both are now improved.